### PR TITLE
Adding latest news block

### PR DESCRIPTION
**Pull request by anuraggupta228**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix <https://github.com/aemsites/momentive/issues/24>

Test URLs:
- Before: https://main--momentive--aemsites.hlx.live/drafts/anurgup/issue-24/latest-news
- After: https://issue-24--momentive--aemsites.hlx.live/drafts/anurgup/issue-24/latest-news

### CODE REVIEW

**Comment by anuraggupta228** [comment]
Also, udpated sidekick Library at : https://issue-24--momentive--aemsites.hlx.page/tools/sidekick/library.html?plugin=blocks&path=/tools/sidekick/blocks/latest-news&index=0

File: blocks/embed/embed.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+  loadScript('https://platform.twitter.com/widgets.js');
+  return embedHTML;
+};
+
+const loadEmbed = (block, link, autoplay) => {
+  if (block.classList.contains('embed-is-loaded')) {
+    return;
+  }
+
+  const EMBEDS_CONFIG = [
```
Comment (samuelopra) [inline blocks/embed/embed.js:63]: The `EMBEDS_CONFIG` array could be moved outside the `loadEmbed` function to avoid recreating it on each function call.

**Comment by samuelopra** [comment]
This looks good, aside from my comment

**Review by samuelopra — APPROVED** [review]
👍

File: blocks/embed/embed.js (hunk 1 above)
Comment (anuraggupta228) [reply blocks/embed/embed.js:63]: this embed block is being imported from https://www.aem.live/developer/block-collection/embed and not touched for  anything specific to the issue implementation pov. And we should forbid us from making any changes into OOTB block code is what i know of (unless we have a very strong reason to do so). So, i’ll have to leave it like that only.  thanks for the review! :)

**Pull request merged**